ingestor-default/src/main/scala/za/co/absa/hyperdrive/ingestor/implementation/manager/checkpoint/CheckpointOffsetManager.scala

```diff
@@ -58,7 +59,7 @@ private[manager] class CheckpointOffsetManager(val topic: String, val checkpoint
 
     if (startingOffsets.isDefined) {
```

Comment:
I would rewrite if statement to something like this: 
`startingOffsets match {
      case Some(startOffset) =>
        logger.info(s"Setting starting offsets for topic '$topic' = $startOffset.")
        streamReader.option(WORD_STARTING_OFFSETS, startOffset)
      case None =>
        logger.info(s"No offsets to set for topic '$topic'.")
        streamReader
    }`

Reply:
Btw you can use triple backticks ``` to make multiline code comments

Reply:
Thanks